### PR TITLE
Adding coursevania.com scraping

### PR DESCRIPTION
**Pull request by earthlydelight**
# Description

Hi, I added support for [Coursevania](https://coursevania.com). I tested it multiple times on my machine and everything seems fine. This changes the aiohttp dependency from `aiohttp` to `aiohttp[speedups]` because the coursevania API returns data compressed with the [Brotli](https://en.wikipedia.org/wiki/Brotli) format.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

Tested it on my machine. The code is really just a copy of the tutorialbar scraper adapted for coursevania.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have added tests that prove my fix is effective or that my feature works (Optional)
- [ ] New and existing unit tests pass locally with my changes (Optional)

### CODE REVIEW

**Comment by fakeid30** [comment]
@earthlydelight thanks for the PR. @cullzie will review this, he is much more rigid than me when it comes to reviewing code 😅

**Comment by cullzie** [comment]
@earthlydelight I will try get around to reviewing this fully during the week. 

Can you re-sync your branch with develop please. There was a new cli arg (`delete_settings`) added in which seems to be removed in your PR

File: udemy_enroller/scrapers/coursevania.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+logger = logging.getLogger("udemy_enroller")
+
+
+class CoursevaniaScraper(BaseScraper):
+    """
+    Contains any logic related to scraping of data from coursevania.com
+    """
+
+    DOMAIN = "https://coursevania.com"
+    AD_DOMAINS = ("https://amzn",)
```
[the remark this answers is not in the document]
Comment (earthlydelight) [reply udemy_enroller/scrapers/coursevania.py]: Removed

File: udemy_enroller/scrapers/coursevania.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        headers = {
+            'Host': 'coursevania.com',
+            'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:85.0) Gecko/20100101 Firefox/85.0',
+            'Accept': 'application/json, text/javascript, */*; q=0.01',
+            'Accept-Language': 'en-US,en;q=0.5',
+            'Accept-Encoding': 'gzip, deflate, br',
+            'X-Requested-With': 'XMLHttpRequest',
+            'Connection': 'keep-alive',
+            'Referer': 'https://coursevania.com/courses/',
+            'Cookie': '__cfduid=d022269be97a5d35b1db7c0195a0a1c091611526580',
```
[the remark this answers is not in the document]
Comment (earthlydelight) [reply udemy_enroller/scrapers/coursevania.py]: I removed the cookie, the request doesn't need it.

File: udemy_enroller/udemy.py [hunk 3]
```diff
@@ -248,7 +248,7 @@ def _check_if_robot(self) -> bool:
         """
         is_robot = True
         try:
-            self.driver.find_element_by_id("px-captcha")
+            self.driver.find_element_by_id("challenge-form")
```
Comment (cullzie) [inline udemy_enroller/udemy.py:251]: This isn't needed anymore. Once we merge in alpha it won't be an issue

**Review by cullzie — APPROVED** [review]

**Comment by fakeid30** [comment]
@cullzie should i merge it?

**Comment by cullzie** [comment]
@fakeid30 I'll merge now and merge back into Alpha

**Comment by cullzie** [comment]
Thanks @earthlydelight for contributing this :rocket:

**Pull request merged**